AndelaOSP/art-dashboard
# [160395201] Fix Data Refetch on Asset Makes and Asset Types Component Pagination

## What does this PR do?
- Fixes the issue with re-fetching data on asset makes component pagination
- Fixes the issue with re-fetching data on asset types component pagination

## Description of Task to be completed?
- It adds a pagination history actions and reducers
- It calls the `loadAssetMake` or `loadAssetTypes` action if the `activePage` is not already in `paginationHistory` store

## How should this be manually tested?
- Start the app
- Navigate to the Asset makes or Asset types page
- Use the pagination and observe how the previously loaded pages are loaded faster

## What are the relevant pivotal tracker stories?
N/A

## Any background context you want to add?
N/A

## Important notes
N/A

## Packages installed
None

## Deployment note
N/A

## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
## Screenshots
N/A
